packages/conf-libuv/conf-libuv.1/opam

```diff
@@ -4,17 +4,20 @@ homepage: "https://github.com/libuv/libuv"
 license: "MIT"
 bug-reports: "https://github.com/libuv/libuv/issues"
 authors: [ "Ryan Dahl" "Bert Belder" "et al." ]
+depends: ["conf-pkg-config"]
 build: [
-  ["pkg-config" "libuv" "--atleast-version=1"] { os != "freebsd" }
-  ["pkgconf" "libuv" "--atleast-version=1"] { os = "freebsd" }
+  ["pkg-config" "libuv" "--atleast-version=1"]
```

Comment:
Yes, it's intentional.
The symlink from pkg-config to pkgconf is created by recent versions of `devel/pkgconf`.
Also `conf-libuv` depends on `conf-pkg-config` now, so `opam depext` can install all dependencies. And `conf-pkg-config` enforces, that freebsd users have pkgconf symlinked as `pkg-config`.

The travis builds fails for the expected reason: Ubuntu precise is too old and I can't distinguish between difference distro versions inside `depexts`.